search/v2/geocoder.cpp

```diff
@@ -213,7 +230,30 @@ Geocoder::~Geocoder() {}
 void Geocoder::SetParams(Params const & params)
 {
   m_params = params;
-  m_retrievalParams = params;
+
+  // Filter stop words.
+  if (m_params.m_tokens.size() > 1)
+  {
+    for (auto & v : m_params.m_tokens)
+    {
+      v.erase(remove_if(v.begin(), v.end(), [] (Params::TString const & s)
+      {
+        return IsStopWord(s);
+      }), v.end());
+    }
+
+    auto & v = m_params.m_tokens;
```

Comment:
v is used as a loop variable over container above and as a container variable here. Please, rename v above to synonyms and v here to tokens.